src/net/sockopt.rs

```diff
@@ -1522,6 +1526,20 @@ pub fn socket_peercred<Fd: AsFd>(fd: Fd) -> io::Result<super::UCred> {
     backend::net::sockopt::socket_peercred(fd.as_fd())
 }
 
+/// `getsockopt(fd, SOL_SOCKET, SO_TXTIME)` — Get transmit timing settings.
+#[cfg(all(target_os = "linux", feature = "time"))]
```

Comment:
should this be `#[cfg(linux_kernel)]`? I don't get the distinction.

Reply:
`linux_kernel` is equivalent to `any(target_os = "linux", target_os = "android"))`. If this feature compiles on Android, then we can enable it there too.